modules/videopress/class.videopress-player.php

```diff
@@ -597,7 +597,16 @@ private function get_flash_parameters() {
 		if ( ! isset( $this->video->players->swf->params ) )
 			return array();
 		else
-			return self::esc_flash_params( apply_filters( 'video_flash_params', (array) $this->video->players->swf->params, 10, 1 ) );
+			return self::esc_flash_params(
+				/**
+				 * Filters the Flash parameters of the VideoPress player.
```

Comment:
I assume we'll just add the hooks to [the list of deprecated hooks](https://github.com/Automattic/jetpack/blob/95e01888e92029cef3db6afb8275f75e0a9bf1c4/class.jetpack.php#L5570) when we remove them? We could add them now if we knew for sure that they'll be removed in 3.7.